tests/base.py

```diff
@@ -0,0 +1,22 @@
+import json
+import os
+import random
+import string
+
+from os.path import expanduser
+
+
+FILE = 'token.json'
+AUTH_TOKEN_PATH = "{}/.evalai/".format(expanduser('~'))
+
+
+class BaseTest:
+
+    @classmethod
+    def setup_class(cls):
+        if not os.path.exists(AUTH_TOKEN_PATH):
+            os.makedirs(AUTH_TOKEN_PATH)
+            with open(AUTH_TOKEN_PATH + FILE, 'w') as fw:
```

Comment:
Please use `os.path` to join paths. This ensures there are no issues on systems like Windows where the path delimiter is `\`.
E.g.
```python
import os.path as osp
with open(osp.join(AUTH_TOKEN_PATH, FILE), 'w') as fw:
```

Starting from Python 3.6, we have a new library called `pathlib` which I personally prefer, but we gotta be backwards compatible. :/